packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -412,30 +412,32 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
         </Stack>
         {showControlBar && !isMobileWithActivePane && (
           <ChatAdapterProvider adapter={chatProps.adapter}>
-            <Stack.Item styles={controlBarContainerStyles}>
-              <CommonCallControlBar
-                callAdapter={callAdapter}
-                chatAdapter={chatProps.adapter}
-                chatButtonChecked={activePane === 'chat'}
-                onChatButtonClicked={toggleChat}
-                peopleButtonChecked={activePane === 'people'}
-                onPeopleButtonClicked={togglePeople}
-                onMoreButtonClicked={onMoreButtonClicked}
-                mobileView={mobileView}
-                disableButtonsForLobbyPage={isInLobbyOrConnecting}
-                /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
-                disableButtonsForHoldScreen={isInLocalHold}
-                callControls={props.callControls}
-                containerHeight={containerHeight}
-                containerWidth={containerWidth}
-                /* @conditional-compile-remove(PSTN-calls) */
-                onClickShowDialpad={alternateCallerId ? onClickShowDialpad : undefined}
-                isCaptionsSupported={isTeamsCall && hasJoinedCall}
-                /* @conditional-compile-remove(video-background-effects) */
-                onShowVideoEffectsPicker={setShowVideoEffectsPane}
-                rtl={props.rtl}
-              />
-            </Stack.Item>
+            <CallAdapterProvider adapter={callAdapter}>
```

Comment:
I don't think we need provider here - in commonCallControlBar we are wrapping everything into CallAdapterProvider. It would be great to reuse that one, and avoid same nested provider

Reply:
Without call adapter provider we can't get the required data for the new components inside control bar, without it I was getting errors